examples/pennylane_quantum_gradients.py

```diff
@@ -0,0 +1,106 @@
+r"""
+.. _quantum_gradients:
+
+Executing Quantum Gradients
+===========================
+
+To do *Quantum Machine Learning* (QML), we need a gradient descent
+strategy. For this, we define a final objective/cost function
+:math:`C` which is to be optimized with respect to some free parameters.
+As explained in :ref:`autograd_quantum` section in the key concepts,
+PennyLane incorporates both analytic differentiation, as well as
+numerical methods (such as the method of finite differences). Both of
+these are done automatically.
+
+PennyLane is the first software library to support **automatic
+differentiation of quantum circuits**. Internally, it uses *classical
+linear combination of unitaries* or “parameter shift” trick to compute
+derivatives, i.e. it evaluates derivatives as the difference of two
+circuit evaluations with shifted parameters.
+
+.. math:: C(\theta_1, \theta_2) = \texttt{circuit}(\theta_1, \theta_2)
+
+.. math:: \partial_{\theta_1} C = a\big[ \texttt{circuit}(\theta_1+s, \theta_2) - \texttt{circuit}(\theta_1 - s, \theta_2) \big]
+
+**NOTE:** The values of the shift and scale parameters :math:`s` and
+:math:`a` typically depend **only** on the *type* of gate, and not its
+location.
+
+By using this method, PennyLane provides a hardware-scalable way to
+compute gradients and to optimize quantum circuits for QML.
+
+Calculating quantum gradients
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let’s initialize a ``device`` and define a Quantum function in a
```

Comment:
Here and elsewhere: maybe use *quantum function*? I think it's either all capital or all small.

Reply:
Okay, I will check all files for this. I find it hard to write quantum with a small q; I have too much respect for this word :D